src/main/java/frc/robot/Constants.java

```diff
@@ -102,9 +104,21 @@ public static final class Swerve {
         public static final NeutralModeValue angleNeutralMode = NeutralModeValue.Coast;
         public static final NeutralModeValue driveNeutralMode = NeutralModeValue.Brake;
 
-        public static final double minElevatorValue = 20;
+        
+        public static final double minElevatorValue = 20; 
         public static final double maxElevatorValue = 500;
 
+        public static final double minElevatorAngle = 29.6;
+        public static final double maxElevatorAngle = 59.9;
+
+        /**
+         * -2.03630363036303630363 was the previous value. it is not equal to maxElevatorAngle/minElevatroAngle but very close, like .01 away.
+         * Where did this value come from?
+         */
+         public static final double degreesToEncoderValue = -(maxElevatorAngle/minElevatorAngle);
```

Comment:
Shouldn't this involve the angle range and max encoder? Like: (maxAngle - minAngle) / maxEncoder